scripts/actions/mobskills/nullsong.lua

```diff
@@ -0,0 +1,37 @@
+-----------------------------------
+-- Nullsong
+-- Description: Removes all beneficial effects from players in an area of effect. Deals darkness damage for each buff removed.
+-- Type: Magical
+-- Utsusemi/Blink absorb: Wipes Shadows
+-- Range: 15' radial
+-- NOTE: Only used if target has 3 or more effects to dispel
+-----------------------------------
+local mobskillObject = {}
+
+mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    if mob:hasStatusEffect(xi.effect.SILENCE) then
+        return 1
+    end
+
+    if target:countEffectWithFlag(xi.effectFlag.DISPELABLE) < 3 then
+        return 1
+    end
+
+    return 0
+end
+
+mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    local count = target:dispelAllStatusEffect(xi.effectFlag.DISPELABLE)
+
+    if count == 0 then
+        skill:setMsg(xi.msg.basic.SKILL_NO_EFFECT)
+        return count
+    end
```

Comment:
If the skill cant fire off without having at least 3 effects to dispel is this check ever going to even get hit?

Reply:
Its an AoE. Other targets may not have any buffs

Reply:
Ah you are right, missed the AOE part